PowerUp/app/src/main/java/powerup/systers/com/StartActivity.java

```diff
@@ -8,17 +8,17 @@
 import android.view.View;
 import android.widget.ImageButton;
 
+
 public class StartActivity extends Activity {
 
+	boolean previouslyStarted;
+	SharedPreferences prefs;
```

Comment:
Don't use abreviations. Calling this `preferences` makes it much easier to read the code in the future